comments/models.py

```diff
@@ -0,0 +1,17 @@
+from django.db import models
+from common.mixins.model import TimestampMixin
+from django.contrib.auth.models import User
+
+# Create your models here.
+
+class Comment(TimestampMixin, models.Model):
+    author = models.ForeignKey(User, on_delete=models.CASCADE)
+    recipe = models.ForeignKey('recipes.Recipe', on_delete=models.CASCADE)
+    text = models.TextField(max_length=2000)
+
+    class Meta:
+        ordering = ['-created_at']
+        verbose_name_plural = 'comments'
+
+    def __str__(self):
+        return f"{self.author}: {self.text[:50]}"
```

Comment:
git sees any file without a newline at the end as an error. You can see the red warning sign here in the code review box. So always make sure to add a newline at the end of any new files you add or work in, so there's always one and only one newline at the end :)